lib/rust/parser/definition/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod parser;
```

Comment:
nl - beforerequesting review, please make sure your code is listed and also make sure you do self-review first - just browse the code on github